packages/dashboard-core-plugins/src/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { Store } from 'redux';
+import Dashboard from '@deephaven/dashboard';
+import {
+  ChartPlugin,
+  ConsolePlugin,
+  FilterPlugin,
+  GridPlugin,
+  LinkerPlugin,
+  MarkdownPlugin,
+  PandasPlugin,
+} from '.';
+
+it('handles mounting and unmount core plugins properly', () => {
+  const store = ({
+    dispatch: jest.fn(() => undefined),
+    getState: jest.fn(() => ({
+      dashboardData: {},
+      workspace: {
+        data: {
+          settings: {},
+        },
+      },
+    })),
+    subscribe: jest.fn(() => undefined),
+  } as unknown) as Store;
```

Comment:
I feel like this should be mocked somewher

Reply:
It is now mocked